workspaces/telepath-queuing-service/package.json

```diff
@@ -6,22 +6,24 @@
   "private": true,
   "license": "MIT",
   "scripts": {
-    "start": "NODE_ENV=production node --require '@babel/register' ./index"
+    "start": "ncc run index.js",
+    "deploy": "now --scope software-concepts --token $NOW_TOKEN && now alias --scope software-concepts --token $NOW_TOKEN"
   },
   "engines": {
     "node": ">=7.6.0"
   },
   "devDependencies": {
     "mockdate": "^2.0.2",
+    "now": "^15.0.0",
     "supertest": "^3.0.0"
   },
   "dependencies": {
-    "@babel/register": "^7.0.0",
+    "@zeit/ncc": "^0.17.0",
```

Comment:
This should be a dev dependency I think?

Reply:
Probably not, because it's used in the 'start' script.